A/abseil_cpp/build_tarballs.jl

```diff
@@ -133,6 +163,7 @@ dependencies = Dependency[
 
 # Build the tarballs, and possibly a `build.jl` as well.
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-               julia_compat="1.6", preferred_gcc_version=v"7")
-
-# Trigger CI: 2
+               augment_platform_block,
+               julia_compat="1.6",
+               preferred_gcc_version=v"7",
+)
```

Comment:
Should lazy artifacts be enabled in order for the platform augmentation to have effect?

Reply:
[No.](https://github.com/JuliaPackaging/BinaryBuilder.jl/blob/bb322ce60a4f8a0a6bf3c657043b47878b9f1785/src/AutoBuild.jl#L1286-L1288)